amaranth/lib/meta.py

```diff
@@ -0,0 +1,109 @@
+import jsonschema
+from abc import abstractmethod, ABCMeta
+from collections.abc import Mapping
+from urllib.parse import urlparse
```

Comment:
Imports are usually put into three groups: stdlib, third party, first party. Like this:

```suggestion
from abc import abstractmethod, ABCMeta
from collections.abc import Mapping

import jsonschema
from urllib.parse import urlparse
```

Reply:
Updated, except urllib which is part of stdlib.